webapp/package.json

```diff
@@ -77,6 +77,7 @@
     "tippy.js": "^4.3.5",
     "tiptap": "~1.25.0",
     "tiptap-extensions": "~1.27.0",
+    "trunc-html": "^1.1.2",
```

Comment:
Very good. You chose the one which has more Github stars than the other two options.

Reply:
hehe, I chose the one we use in the `excerptMiddleware`, but whoever chose it must have looked at the GitHub stars :smile_cat: